update-firefox-version

```diff
@@ -22,36 +22,75 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-try:
-    from bs4 import BeautifulSoup
-except ImportError:
-    print('Please install BeautifulSoup! (e.g., `sudo pip3 install bs4`)')
-    exit(1)
+from html.parser import HTMLParser
 
-from collections import OrderedDict
+import collections
 import hashlib
 import json
-import logging
 import os
+import os.path
+import sys
 import urllib.error
-import urllib.parse
 import urllib.request
 
-CURRENT_VERSION = '52.6.0esr'
 
-class FirefoxJsonGenerator():
-    JSON_FILE_NAME = 'org.mozilla.Firefox.json.in'
-    MOZILLA_ARCHIVE_URL = 'https://archive.mozilla.org/pub/firefox/releases/{version}/linux-x86_64/'
+CURRENT_VERSION = '60.7.1esr'
```

Comment:
I checked with @ramcq / @wjt and we agreed to go with non-esr version here, so please update. Current latest is `68.0.1`.